docs/markdown/snippets/intl-dependency.md

```diff
@@ -0,0 +1,35 @@
+## New custom dependency for libintl
+
+Meson can now find the library needed for translating messages via gettext.
+This works both on systems where libc provides gettext, such as GNU or musl,
+and on systems where the gettext project's standalone intl support library is
+required, such as macOS.
+
+Rather than doing something such as:
+
+```
+intl_dep = dependency('', required: false)
+
+if cc.has_function('ngettext')
+  intl_found = true
+else
+  intl_dep = cc.find_library('intl', required: false)
+  intl_found = intl_dep.found()
+endif
+
+if intl_found
+  # build options that need gettext
+  # conf.set('ENABLE_NLS', 1)
```

Comment:
Why should this line be commented out?

Reply:
Because it made perfect sense at 1AM!

... You're right. That looks odd.

Reply:
Fixed